pkg/vm/memory/memory_value.go

```diff
@@ -93,16 +94,16 @@ func (address MemoryAddress) String() string {
 //   - or a pointer to another Memory Cell (a `MemoryAddress`)
 type MemoryValue struct {
```

Comment:
I would add a comment here explaining that memory address will be contained inside a `felt`

Reply:
done